doc/tutorials/test-templates-basic.md

```diff
@@ -0,0 +1,306 @@
+# Creating Tests with JSonnet Templates
+
+## Prerequisites
+
+1. A clone of this repository.
+1. Build dependencies from [developing.md](/doc/developing.md).
+1. A GKE cluster with GPUs and (optionally) Cloud TPUs. Accelerator availability depends on GCE zone. All of the accelerators used in this tutorial are available in `us-central1-b`.
+  - Example command:
+    ```
+      gcloud beta container clusters create tutorial-cluster \
+        --zone us-central1-b \
+        --release-channel regular \
+        --machine-type n1-standard-4 \
+        --accelerator "type=nvidia-tesla-v100,count=1" \
+        --scopes "https://www.googleapis.com/auth/cloud-platform" \
+        --num-nodes 1 \
+        --enable-ip-alias \
+        --enable-autoupgrade \
+        --enable-tpu
+      kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded.yaml
+    ```
+1. A GCS bucket.
+  - To create a new GCS bucket, run `gsutil mb -c standard -l us-central1 gs://your-bucket-name`
+
+## Creating a simple test without accelerators
+
+We'll start by creating a simple test that runs the example MNIST model from the TensorFlow model garden on CPUs. First, build the TensorFlow test image from this repository:
+
+```bash
+gcloud builds submit --config images/tensorflow/cloudbuild.yaml --project=xl-ml-test --substitutions _VERSION=r2.2,_BASE_IMAGE_VERSION=2.2.0,_MODEL_GARDEN_BRANCH=r2.2.0
```

Comment:
I'd make this an env variable. People like to copy-paste without reading first :)

Reply:
Good idea. Done.